src/CSET/operators/ensembles.py

```diff
@@ -0,0 +1,117 @@
+# © Crown copyright, Met Office (2022-2024) and CSET contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A module containing different diagnostics for ensembles.
+
+The diagnostics here are applicable to ensembles and apply generally to
+ensembles. They are not just limited to considering ensemble spread.
+"""
+
+import iris
+import numpy as np
+
+
+def DKE(
+    u: iris.cube.Cube,
+    v: iris.cube.Cube,
+) -> iris.cube.Cube:
+    r"""Calculate the Difference Kinetic Energy (DKE).
+
+    Parameters
+    ----------
+    u: iris.cube.Cube
+        Iris cube of the u component of the wind field for all members. Must
+        include a realization coordinate; realization must be the first
+        coordinate.
+    v: iris.cube.Cube
+        Iris cube of the v component of the wind field same format as u.
+
+    Returns
+    -------
+    DKE: iris.cube.Cube
+        An iris cube of the DKE for each of the control - member comparisons.
+
+    Notes
+    -----
+    The Difference Kinetic Energy, or DKE was first used in an ensemble sense
+    by [Zhangetal2002]_. It is calculated as
+    .. math:: \frac{1}{2}u'u' + \frac{1}{2}v'v'
+    where
+    .. math:: x' = x_{control} - x_{perturbed}
+    for each member of the ensemble.
+
```

Comment:
```suggestion
    by [Zhangetal2002]_. It is calculated as

    .. math:: \frac{1}{2}u'u' + \frac{1}{2}v'v'

    where

    .. math:: x' = x_{control} - x_{perturbed}

    for each member of the ensemble.

```
Newlines are needed around maths for it to render properly.